pkgs/by-name/bs/bs-manager/wine-inherit-env.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/main/services/mods/bs-mods-manager.service.ts b/src/main/services/mods/bs-mods-manager.service.ts
+index 5f72609..c842fda 100644
+--- a/src/main/services/mods/bs-mods-manager.service.ts
++++ b/src/main/services/mods/bs-mods-manager.service.ts
+@@ -174,6 +174,7 @@ export class BsModsManagerService {
+                 throw new CustomError("Could not find BSManager WINEPREFIX path", "no-wineprefix");
+             }
+             env.WINEPREFIX = winePrefix;
++            Object.assign(env, process.env);
```

Comment:
Could you explain why this was needed? I have a guess, but I'd still like to see an explanation.